templates/compose/vendor.yaml.erb

```diff
@@ -1,5 +1,7 @@
 version: '3.6'
 
+<%- @name = @config['app']['name'] -%>
```

Comment:
You could've just declared this var in the Rakefile instead of copying it to every template

Reply:
okay